Backend/KFC-WebAPI/Scripts/README.md

```diff
@@ -0,0 +1,219 @@
+<!-- IGNORE THE HTML BLOCK BELOW, THE INTERESTING PART IS AFTER IT -->
```

Comment:
This has been removed by the bug branch. Accept the changes from the bug branch.

Reply:
I merged the branch 2 days ago. I don't have anymore

Reply:
on that commit, you accepted some stuff but still seems like you kept old files.